rand_test.go

```diff
@@ -12,3 +15,19 @@ func TestRand(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestAllocations(t *testing.T) {
+	n := int(testing.AllocsPerRun(10, func() {
+		buf := make([]byte, 32)
+		openssl.RandReader.Read(buf)
+		sink ^= buf[0]
```

Comment:
Seems a little odd to use the definition from pbkdf2_test.go... maybe move `sink` to openssl_test.go and explicitly mention that it's for everyone to use?

Reply:
Good point. Done.